extensions/wikia/Recirculation/js/helpers/recommendedContent.js

```diff
@@ -0,0 +1,107 @@
+/*global define*/
+define('ext.wikia.recirculation.helpers.recommendedContent', [
+    'jquery',
+    'wikia.window',
+    'wikia.nirvana',
+    'ext.wikia.recirculation.helpers.blacklist',
+    'wikia.log',
+    'wikia.eventLogger'
+], function ($, w, nirvana, blacklist, log, eventLogger) {
+    'use strict';
+
+    var numberOfArticleFooterSlots = $('#mixed-content-footer').data('number-of-wiki-articles');
+    var REQUEST_ID;
```

Comment:
I thought capses are reserved for consts. How about using a regular camelcase for a variable?